test/html/scrollToElement.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>scrollToElement - Test</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <style>
+      
+      #second {
+        margin: 250px;
+        height: 800px;
+        width: 2000px;
+        background-color: coral;
+      }
+
+    </style>
+  </head>
+  <body>
+    <h1>scrollToElement</h1>
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
+    <h1 id="first">
+      <a id="first-a" href="#" onclick="scrollFunction('second');return false;">Click me to scroll to second element
+        <p id="first-clue"></p></a>
+    </h1>
+    .<br />.<br />.<br />.<br />.<br />.<br />.<br />.<br />
```

Comment:
I like the test, but could you please replace the `<br>` with a CSS class that you apply to elements that you want to keep space between:

```css
.spacer {
  margin-bottom: 500px;
}
```
This way we can keep the presentation details separate from the markup.